apps/web/src/pages/WorkStreams/CreateWorkStreamPage.tsx

```diff
@@ -0,0 +1,309 @@
+                type="text"
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+                aria-describedby="plainLanguageNameHelp"
+              />
+              <Input
+                id="plainLanguageName.fr"
+                label={intl.formatMessage({
+                  defaultMessage: "Plain language alternative (French)",
+                  id: "OKCVhm",
+                  description: "Label for plain language alt french input",
+                })}
+                name="plainLanguageName.fr"
+                type="text"
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+                aria-describedby="plainLanguageNameHelp"
+              />
+              <div data-h2-grid-column="p-tablet(span 2)">
+                <Select
+                  id="community"
+                  name="community"
+                  label={intl.formatMessage(adminMessages.community)}
+                  nullSelection={intl.formatMessage(
+                    commonMessages.selectACommunity,
+                  )}
+                  rules={{
+                    required: intl.formatMessage(errorMessages.required),
+                  }}
+                  options={communityOptions}
+                />
+              </div>
+            </div>
+            <CardSeparator />
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column) p-tablet(row)"
+              data-h2-gap="base(x1)"
+              data-h2-align-items="base(center)"
+            >
+              <Submit
+                text={intl.formatMessage({
+                  defaultMessage: "Create work stream",
+                  id: "RjKlKz",
+                  description: "Text to create a work stream",
+                })}
+              />
+              <Link
+                mode="inline"
+                color="warning"
+                href={paths.workStreamTable()}
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Cancel and go back to work streams",
+                  id: "kh9KTx",
+                  description: "Link text to cancel updating a work stream",
+                })}
+              </Link>
+            </div>
+          </BasicForm>
+        </CardBasic>
+      </Hero>
+    </>
+  );
+};
+
+const CreateWorkStream_Query = graphql(/* GraphQL */ `
+  query CreateWorkStreamOptions {
+    communities {
+      id
+      name {
+        en
+        fr
+        localized
+      }
+    }
+  }
+`);
+
+const CreateWorkStreamPage = () => {
+  const [{ data: lookupData, fetching, error }] = useQuery({
+    query: CreateWorkStream_Query,
+  });
+
+  const communityOptions: OptGroupOrOption[] =
+    unpackMaybes(lookupData?.communities).map(({ id, name }) => ({
+      value: id,
+      label: name?.localized,
+    })) ?? [];
```

Comment:
Sorry, I was not entirelty clear. You do not need the `?? []` here with `unpackMaybes` because it is part of the function itself. 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/145ac541351efca06d37d08d969469fb2a65c865/packages/helpers/src/utils/util.ts#L171-L173